arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json

```diff
@@ -1252,6 +1292,49 @@
         }
       }
     },
+    "RegistryUsageListResult": {
```

Comment:
How large is this list? Consider using `x-ms-pageable` if the page is large enough (>50 entries)